### PR TITLE
337 Fixed Scipy import deprecation warning

### LINKED ISSUE #337: Fix deprecation in ahi_hsd.py: Please use `zoom` from the `scipy.ndimage` namespace, the `scipy.ndimage.interpolation` namespace is deprecated.
# Requested Update

### Description
Importin the `ahi_hsd` reader causes the following deprecation warnings to be raised:

```
/local/home/jsolbrig/geoips/github/geoips/geoips/plugins/modules/readers/abi_netcdf.py:22: DeprecationWarning: Please use `zoom` from the `scipy.ndimage` namespace, the `scipy.ndimage.interpolation` namespace is deprecated.
  from scipy.ndimage.interpolation import zoom
/local/home/jsolbrig/geoips/github/geoips/geoips/plugins/modules/readers/ahi_hsd.py:25: DeprecationWarning: Please use `zoom` from the `scipy.ndimage` namespace, the `scipy.ndimage.interpolation` namespace is deprecated.
  from scipy.ndimage.interpolation import zoom
```

We should update accordingly.

### Background and Motivation
*  [ OPTIONAL: Provide additional detailed information and background related to this issue ]

### Alternative Solutions
*  [ OPTIONAL: If appropriate, list alternative fixes that could be considered]
  
### Environment
*  [ IF APPROPRIATE: Corresponding repository versions ]

### Code to demonstrate issue
*  [ IF APPROPRIATE: Can include steps here to identify/demonstrate bug fix or desired feature ]

### Checklist for Completion
* [ ]

### PR DESCRIPTION
**Pull request by evrose54**
# Reviewer Checklist

* [x] Required ***existing tests*** pass (ie full_test.sh, others as appropriate)
  * [x] None required (explain if checked, delete line if not checked)
* [x] Required ***unit tests*** added and pass for new/modified functionality
  * [x] None required (explain if checked, delete line if not checked)
* [x] Required ***integration tests*** added and pass for new/modified functionality
  * [x] None required. (explain if checked, delete line if not checked)
* [x] Required ***documentation*** added for new/modified functionality
  * [x] None required (explain if checked, delete line if not checked)
* [x] Required ***release notes*** added for new/modified functionality
  * [x] None required. (explain if checked, delete line if not checked)

https://github.com/NRLMMD-GEOIPS/.github/blob/main/.github/review-template.md

# Related Issues
fixed #337 

# Testing Instructions
Run geoips/tests/integration_tests/full_test.sh to prove that no errors have occurred.

# Summary
This PR has been created to update the 'from scipy.ndimage.interpolation import zoom' to 'from scipy.ndimage import zoom', as the interpolation library was deprecated. No problems exist anymore. This PR also adds the new v1_11_5a0.rst to the release docs.

  - modified: geoips/geoips/plugins/modules/readers/abi_netcdf.py
  - modified: geoips/geoips/plugins/modules/readers/ahi_hsd.py
  - added: geoips/docs/source/releases/v1_11_5a0.rst

### CODE REVIEW

**Review by mindyls — DISMISSED** [review]
I added that the deprecation happened Dec 2018.

Technically I suppose we should put scipy>=1.2 in pyproject.toml, but is it really necessary when it is a 5 year old version?

File: docs/source/releases/v1_11_5a0.rst [hunk 1]
```diff
@@ -17,6 +17,22 @@ Version 1.11.5a0 (2023-09-27)
...
 * Remove flake8_docstring_only option from check_code.sh workflow call
+* Fixed from scipy.ndimage.interpolation import zoom deprecation warning
+
+Bug Fixes
+=========
+
+Fixed from scipy.ndimage.interpolation import zoom deprecation warning
+----------------------------------------------------------------------
+
+The scipy.ndimage.interpolation library was deprecated as of scipy v1.2 (Dec 2018),
```
Comment (mindyls) [inline docs/source/releases/v1_11_5a0.rst:28]: Is it correct practice to add scipy>=1.2 to pyrpoject.toml?  I'm ok without adding a 5 year old version to pyproject.toml if others are.

Comment (mindyls) [reply docs/source/releases/v1_11_5a0.rst]: I went ahead and added scipy > 1.2 to pyproject.toml.  Resolve if you are comfortable with this, or I can back it out.

**Review by mindyls — APPROVED** [review]
As soon as someone resolves my conversation, this is good to go.

Adding scipy>1.2 seems stupid, but it won't hurt anything.

**Review by jsolbrig — APPROVED** [review]

**Review by mindyls — APPROVED** [review]
Looks good to me.

**Review by jsolbrig — APPROVED** [review]

**Pull request merged**